### PR TITLE
Renamed webots.exe to webots-bin.exe

### LINKED ISSUE #137: Unable to Load JPEG Textures from cmd.exe
**Describe the Bug**
JPEG textures doesn't load and produce the following error message:
```
WARNING: Texture image size of 'C:/Program Files/Webots/projects/objects/obstacles/protos/textures/oil_barrel_bottom_occlusion.jpg' is not a power of two: rescaling it from -1x-1 to 0x0.
WARNING: OilBarrel (PROTO) > Shape > PBRAppearance > ImageTexture: Cannot load texture 'C:/Program Files/Webots/projects/objects/obstacles/protos/textures/oil_barrel_bottom_occlusion.jpg': Unsupported image format.
```

This bug was reported by several users (e.g. #326) and I was finally able to reproduce it by starting Webots from `cmd.exe`. It is working fine when launched from the desktop shortcut.

**Steps to Reproduce**
1. Launch Webots with `cmd.exe`

**Expected behavior**
The textures should load without any problem.

**Screenshots**
capture

**System**
 - Operating System: Windows 10 (only from cmd.exe)
 - `PATH` environmental variable from `cmd.exe`:
```
C:\Program Files\Python37\Scripts\;C:\Program Files\Python37\;C:\msys64\home\David\develop\webots-develop\webots\msys64\mingw64\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\DLR\Sumo\bin\;C:\Program Files (x86)\DLR\Sumo\tools\;C:\Program Files\MATLAB\R2018a\bin;C:\Program Files\MATLAB\R2017a\runtime\win64;C:\Program Files\MATLAB\R2017a\bin;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\Git LFS;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Python37\Scripts\;C:\Python37\;C:\msys64\home\David\develop\webots-develop\webots\msys64\mingw64\bin;C:\Users\David\AppData\Local\Microsoft\WindowsApps;C:\Users\David\AppData\Local\atom\bin;C:\Users\David\AppData\Local\Microsoft\WindowsApps;C:\Python27;C:\Program Files\Java\jdk1.8.0_121\bin;
```

### PR DESCRIPTION
**Pull request by omichel**
Fixes #137 as it was reported again by several users.
This fix simply renames `webots.exe` to `webots-bin.exe`.
It fixes the problem of typing `webots` in a DOS CMD console.
Previously, this was launching `webots.exe` directly and was causing problems with JPEG textures as explained in #137.
Now, it launches a different `webots.exe` which is a simple launcher for `webots-bin.exe`.
`webots.exe` sets the `PATH` environment variable appropriately and starts `webots-bin.exe`.
As a consequence, Webots launches nicely and the JPEG textures bug doesn't show up any more.
It will still show up if the user type `webots-bin` in the DOS CMD console, which is less likely.

### CODE REVIEW

**Comment by omichel** [comment]
The `webots.bat` launcher is not a good solution to start Webots from the icon because it opens a DOS console in the background, which doesn't look nice. I rewrote this `webots.bat` in C to produce a `webots.exe` which does basically the same thing, except it doesn't open this ugly DOS console.

**Comment by DavidMansolino** [comment]
:eyes:

File: src/install_scripts/bash_profile.windows [hunk 1]
```diff
@@ -46,3 +46,5 @@ MSYS2_WINDOWS_HOME=`cygpath -w /`
 export MSYS2_HOME=${MSYS2_WINDOWS_HOME::-1}
 
 export LANG=en_US.UTF-8  # needed on some locales to avoid floating point representation issues
+
+alias webots=webots-bin.exe
```
[the remark this answers is not in the document]
Comment (omichel) [reply src/install_scripts/bash_profile.windows:50]: Yes, I was also not sure about that. I think it is safer to remove it indeed. However, when debugging with gdb, we should take care of launching `webots-bin.exe` directly.

**Comment by omichel** [comment]
That's odd... It was working for me... Let me try to reproduce it...

**Comment by omichel** [comment]
I found the problem. It was working just before I added the `-mwindows` link option which disables the DOS console. The `-mwindows` flags actually somehow breaks the `argv[0]` value on which I relied to compute the new `PATH`. So, I ended up with two different launchers like in Python or Java (with `python.exe` / `pythonw.exe` and `java.exe` / `javaw.exe`).

**Comment by DavidMansolino** [comment]
👀

File: src/webots/Makefile [hunk 2]
```diff
@@ -712,7 +724,7 @@ webots.o:gui/webots.rc gui/webots.ico gui/manifest.xml
 
 clean:
 	@echo "# cleaning"
-	@rm -rf $(TARGET) $(FILES_TO_REMOVE) build
+	@rm -rf $(TARGET) $(FILES_TO_REMOVE) $(LAUNCHER) build
```
Comment (DavidMansolino) [inline src/webots/Makefile:727]:
```suggestion
	@rm -rf $(TARGET) $(FILES_TO_REMOVE) $(LAUNCHER) $(LAUNCHERW) build
```

**Comment by omichel** [comment]
Thank you for the review! It seems copying the `libjpeg-8.dll` as we do for the `zlib1.dll` is not needed for me (it works for me without that). But removing `zlib.dll` doesn't break my installation either. However, I fear that it may be needed in some user environments as I could not reproduce the exact same bug some users originally reported. So, it is an additional security (in case a bad `libjpeg-8.dll` exists in the `System32` folder of Windows). We will see if users stop reporting this error once revision 2 (or R2020b) is released.

**Pull request merged**